physical_infra_provider_configuration/automation/topics/creating_instance_lenovo_Xclarity_class.adoc

```diff
@@ -0,0 +1,41 @@
+==== Creating a new instance to the LenovoXclarity class
+
+1.  Select your domain created and drill down Your Domain name→ System → Event → EmsEvent →LenovoXclarity and select **(LenovoXclarity)**.
+
+2.  Click **(Configuration)**, then click **(Add a new Instance)**.
+
+3.  Enter with the name
+
+    a.  The name must be equal to event name that will be managed, because the Cloud Forms match the instance with the event by the name.
+    
+    b.  Find the name of the event that you wish in this link **<TO DO>**
```

Comment:
@CharlleDaniel Did you intend to leave this TO DO placeholder?

Reply:
So, is missing a documentation about the event's name, Lucas is working in a new document about possibles event's name in the LXCA and I'm waiting the link  for this document.

Reply:
@CharlleDaniel Everything looks good. Is it OK if I go ahead and merge these changes? You can always create a PR later to update the link that depends on Lucas's document.

Reply:
@skovic no problem, go ahead. Thank you. :+1: